rmw_fastrtps_cpp/src/rmw_publisher.cpp

```diff
@@ -168,6 +168,12 @@ rmw_create_publisher(
   rmw_publisher->implementation_identifier = eprosima_fastrtps_identifier;
   rmw_publisher->data = info;
   rmw_publisher->topic_name = reinterpret_cast<char *>(rmw_allocate(strlen(topic_name) + 1));
+
+  if (!rmw_publisher->topic_name) {
+    RMW_SET_ERROR_MSG("failed to allocate memory for publiser node name");
```

Comment:
Shoulder be "publisher topic name" instead of "pubiser node name".

Similar below.

Reply:
done. thanks.